src/intl/translations/index.ts

```diff
@@ -4,11 +4,13 @@ import { en } from './en';
 import { es } from './es';
 import { fr } from './fr';
 import { ja } from './ja';
+import { pt_br } from './pt-br';
 import { TranslationLanguage } from './types';
 import { zh } from './zh';
 
 export * from './types';
 
+//@ts-ignore
```

Comment:
this is temporary and will be removed in https://github.com/GitbookIO/gitbook/pull/2317